plugin/trino-mysql/src/test/java/io/trino/plugin/mysql/TestMySqlTypeMapping.java

```diff
@@ -481,6 +482,109 @@ public void testDate()
         }
     }
 
+    @Test
+    public void testTime()
+    {
+        SqlDataTypeTest.create()
```

Comment:
We should test this with different session zones too?

Reply:
@hashhar Updated.